src/foundry/client/pixi/layers/placeables/lighting.d.ts

```diff
@@ -0,0 +1,141 @@
+     * @param darkness - The new darkness level
+     * @param prior    - The prior darkness level
+     */
+    protected _onDarknessChange(darkness: number, prior: number): void;
+
+    /**
+     * @deprecated since v10, will be removed in v12
+     * @remarks "LightingLayer#background has been refactored to EffectsCanvasGroup#background"
+     */
+    get background(): EffectsCanvasGroup["background"];
+
+    /**
+     * @deprecated since v10, will be removed in v12
+     * @remarks "LightingLayer#illumination has been refactored to EffectsCanvasGroup#illumination"
+     */
+    get illumination(): EffectsCanvasGroup["illumination"];
+
+    /**
+     * @deprecated since v10, will be removed in v12
+     * @remarks "LightingLayer#channels has been refactored to EffectsCanvasGroup#lightingChannelColors"
+     * @remarks lightingChannelColors was removed in v11 without this deprecation being updated
+     */
+    get channels(): undefined;
+
+    /**
+     * @deprecated since v10, will be removed in v12
+     * @remarks "LightingLayer#coloration has been refactored to EffectsCanvasGroup#coloration"
+     */
+    get coloration(): EffectsCanvasGroup["coloration"];
+
+    /**
+     * @deprecated since v10, will be removed in v12
+     * @remarks "LightingLayer#darknessLevel has been refactored to Canvas#darknessLevel"
+     */
+    get darknessLevel(): Canvas["darknessLevel"];
+
+    /**
+     * @deprecated since v10, will be removed in v12
+     * @remarks "LightingLayer#globalLight has been refactored to CanvasIlluminationEffects#globalLight"
+     */
+    get globalLight(): CanvasIlluminationEffects["globalLight"];
+
+    /**
+     * @deprecated since v10, will be removed in v12
+     * @remarks "LightingLayer#sources has been refactored to EffectsCanvasGroup#lightSources"
+     */
+    get sources(): EffectsCanvasGroup["lightSources"];
+
+    /**
+     * @deprecated since v10, will be removed in v12
+     * @remarks "LightingLayer#version has been refactored to EffectsCanvasGroup#lightingVersion"
+     * @remarks lightingVersion was removed in v11 without this deprecation being updated
+     */
+    get version(): undefined;
+
+    /**
+     * @deprecated since v10, will be removed in v12
+     * @remarks "LightingLayer#activateAnimation has been refactored to EffectsCanvasGroup#activateAnimation"
+     */
+    activateAnimation(): void;
+
+    /**
+     * @deprecated since v10, will be removed in v12
+     * @remarks "LightingLayer#deactivateAnimation has been refactored to EffectsCanvasGroup#deactivateAnimation"
+     */
+    deactivateAnimation(): void;
+
+    /**
+     * @deprecated since v10, will be removed in v12
+     * @remarks "LightingLayer#animateDarkness has been refactored to EffectsCanvasGroup#animateDarkness"
+     */
+    animateDarkness(
+      ...args: Parameters<EffectsCanvasGroup["animateDarkness"]>
+    ): ReturnType<EffectsCanvasGroup["animateDarkness"]>;
```

Comment:
I think you can do `animateDarkness: EffectCanvasGroup["animateDarkness"]` with no observable difference. If you can find a way to distinguish the two at the type level I'd be surprised. `{ foo: () => number } extends { foo(): number }` and so on.